app/templates/view_buyers.html

```diff
@@ -100,6 +100,18 @@
         <textarea rows="5" name="edit_seller_email_list" id="edit_seller_email_list" class="text-area">{{seller_email_list}}</textarea>
         <input type="submit" value="Save" class="button-save">
     </form>
+    <form action="{{ url_for('.update_brief',brief_id=brief_id) }}" method="post" class="question">
+        {{ form.csrf_token }}
+        <label class="question-heading" for="edit_area_of_expertise">Select area of expertise</label>
+        <select name="edit_area_of_expertise" id="edit_area_of_expertise">
+            <option value="">None Selected</option>
+            {% for _ in area_of_expertise_list %}
```

Comment:
a multi line for loop may not be the best place for short variable names like `_`

Reply:
changed it to `i`